### PR TITLE
refactor: [M3-8648] – Migrate `Paper` to `ui` package

### PR DESCRIPTION
**Pull request by dwiley-akamai**
## Description 📝
Migrate `Paper` from `manager` to `ui` package.

## Target release date 🗓️
11/12/24

### Verification steps
- All tests should pass
- There should be no adverse impacts observed in Cloud

## As an Author I have considered 🤔
- [X] 👀 Doing a self review
- [X] ❔ Our [contribution guidelines](https://github.com/linode/manager/blob/develop/docs/CONTRIBUTING.md)
- [X] 🤏 Splitting feature into small PRs
- [X] ➕ Adding a changeset
- [X] 🧪 Providing/Improving test coverage
- [X] 🔐 Removing all sensitive information from the code and PR description
- [X] 🚩 Using a feature flag to protect the release
- [X] 👣 Providing comprehensive reproduction steps
- [X] 📑 Providing or updating our documentation
- [X] 🕛 Scheduling a pair reviewing session
- [X] 📱 Providing mobile support
- [X] ♿  Providing accessibility support

### CODE REVIEW

File: packages/ui/src/utilities/omittedProps.ts [hunk 1]
```diff
@@ -24,7 +24,10 @@ export const omittedProps = <Props>(props: Array<keyof Props>) => (
  * @param toRemove Array of props to remove
  * @returns Object with props removed
  */
-export const omitProps = <Props extends {}, Keys extends keyof Props & string>(
+export const omitProps = <
+  Props extends NonNullable<unknown>,
+  Keys extends keyof Props & string
+>(
```
Comment (dwiley-akamai) [inline packages/ui/src/utilities/omittedProps.ts:30]: This change is due to this:

Screenshot 2024-10-29 at 10 49 15 AM

I went with the option that seemed to best maintain parity, but we can revisit if necessary.

**Comment by dwiley-akamai** [comment]
Putting this back into draft status until the errors related to `omittedProps` are worked through

File: packages/ui/src/utilities/omittedProps.ts
Comment (hkhalil-akamai) [inline packages/ui/src/utilities/omittedProps.ts:1]: Should we move `omittedProps.test.tsx` along with this file?

**Review by hkhalil-akamai — COMMENTED** [review]
Other comments:
* Unit test failure appears unrelated
* We should update the import from the example in [`02-component-structure.md:L20`](https://github.com/linode/manager/blob/b1f2b249e086d610910b403e1274ebd62f6dbfe6/docs/development-guide/02-component-structure.md?plain=1#L20)

**Review by harsh-akamai — APPROVED** [review]
- Paper in Cloud ✅ 
- Code Review ✅ 

Thanks for the changes @dwiley-akamai! 🚀

**Review by bnussman-akamai — APPROVED** [review]

**Pull request merged**